src/SDK.ts

```diff
@@ -1369,4 +1408,10 @@ export default class SDK implements ISDK {
       headers[OmnichannelHTTPHeaders.ocSessionId] = sessionId;
     }
   }
+
+  private seRequestIdHeader = (requestId: string | undefined, headers: any) => {
```

Comment:
eslint for any

Reply:
We have `private setSessionIdHeader()`. This can be reused in some way

Reply:
typo: `seRequestIdHeader`